portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,18 +23,16 @@
 import java.util.ArrayList;
 
 /** Servlet that returns some hard coded strings. */
-@WebServlet("/data")
+@WebServlet("/comment")
 public class DataServlet extends HttpServlet {
+  private final String CONTENT_TYPE = "application/json";
+  private String stringList = new Gson()
+      .toJson(new String[] {"test str1", "test str2", "test str3"});
```

Comment:
Just the declaration and instantiation goes up here unless this is a constant.

Reply:
Technically it is a constant (Oh I didn't use uppercase).
 I'll delete it with values from datastore later.
Anyways I've moved the value assignment to constructor.